src/test/java/org/openmainframeproject/cobolcheck/ExpanderTest.java

```diff
@@ -140,6 +236,144 @@ public void it_inserts_a_mock_correctly() throws IOException {
+                    "       UT-BEFORE-EACH.                                                          " + Constants.NEWLINE +
+                    "      *****************************************************************         " + Constants.NEWLINE +
+                    "      *This is performed before each Test Case                                  " + Constants.NEWLINE +
+                    "      *****************************************************************         " + Constants.NEWLINE +
+                    "           CONTINUE                                                             " + Constants.NEWLINE +
+                    "           .                                                                    " + Constants.NEWLINE +
+                    "                                                                                 " + Constants.NEWLINE +
+                    "       UT-AFTER-EACH.                                                           " + Constants.NEWLINE +
+                    "      *****************************************************************         " + Constants.NEWLINE +
+                    "      *This is performed after each Test Case                                   " + Constants.NEWLINE +
+                    "      *****************************************************************         " + Constants.NEWLINE +
+                    "           CONTINUE                                                             " + Constants.NEWLINE +
+                    "           .                                                                    " + Constants.NEWLINE +
+                    "                                                                                 " + Constants.NEWLINE +
+                    "       UT-INITIALIZE-MOCK-COUNT.                                                " + Constants.NEWLINE +
+                    "      *****************************************************************         " + Constants.NEWLINE +
+                    "      *Sets all global mock counters and expected count to 0                    " + Constants.NEWLINE +
+                    "      *****************************************************************         " + Constants.NEWLINE +
+                    "           CONTINUE                                                             " + Constants.NEWLINE +
+                    "           .                                                                    " + Constants.NEWLINE +
+                    "                                                                                 " + Constants.NEWLINE +
+                    "      *CALL \"PROGRAM\" USING VALUE-1 ON EXCEPTION PERFORM 100-WELCOME END      " + Constants.NEWLINE +
+                    "      * -CALL.                                                                  " + Constants.NEWLINE +
```

Comment:
I do not like the way that the entire call statement is concatenated into a string, and then split over the required number of lines.
I would prefer that the comment looks something like this, that preserves the original lines and spacing:
```
* CALL \"PROGRAM\" USING VALUE-1
*   ON EXCEPTION
*   PERFORM 100-WELCOME
* END-CALL.
```
If you can find an easy fix for that, it would be awesome :-D

Reply:
@Rune-Christensen 
Thanks for raising this issue.
I have added code for this in my latest PR.
The commit link is : [add pretty format for multiline comment statements](https://github.com/openmainframeproject/cobol-check/pull/312/commits/b7516b04c25dfe8075b50968c51c387bd7cc51a8)